### PR TITLE
Change password echos

### PR DESCRIPTION
**Pull request by JaBurd**
Change the password echos to be passed as a group and individually.

Resolves issue #44 by splitting up the passwords to be passed individually vs a single echo.

Also the `test -e` not_if clause was causing issues as `test -e` didn't work properly, reverted to a file exists test.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## v2.3.1
+  Update how the passwords are sent to kdb5_util when creating realm db
+  Changed guard for realm db creation from test -e to File.exist?
+
```
Comment (wolf31o2) [inline CHANGELOG.md]: Put this under Unreleased, please

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs and configures Kerberos V authentication'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.3.0'
+version          '2.3.1'
```
Comment (wolf31o2) [inline metadata.rb]: Don't bump the version, please. I'll do that when I release a new version to Supermarket.

File: recipes/kadmin_init.rb [hunk 3]
```diff
@@ -25,18 +25,18 @@
 log 'create-krb5-db' do
   message 'Creating Kerberos Database... this may take a while...'
   level :info
-  not_if "test -e #{node['krb5']['kdc_conf']['realms'][default_realm]['database_name']}"
+  not_if { ::File.exist?(node['krb5']['kdc_conf']['realms'][default_realm]['database_name']) }
```
Comment (wolf31o2) [inline recipes/kadmin_init.rb]: This causes tests to pass by accident. If you run the tests on a machine which includes these files locally, it will fail. Why did you move away from the shell-based `test -e`?

File: metadata.rb (hunk 2 above)
Comment (JaBurd) [reply metadata.rb]: apologies, wasn't sure if I needed to add that or not.

File: CHANGELOG.md (hunk 1 above)
Comment (JaBurd) [reply CHANGELOG.md]: apologies, wasn't sure if I needed to add that or not.

File: recipes/kadmin_init.rb (hunk 3 above)
Comment (JaBurd) [reply recipes/kadmin_init.rb]: in our testing with kitchen-ec2 it was always skipping these steps.  I just re-tested and it worked as expected with the `test -e`, reverting.

**Comment by wolf31o2** [comment]
Thanks for the submission! I'm going to merge this and it will go out with the next release.
